docs/guides/ultimate-guides/torrents/ultimate-intro-to-torrents/index.md

```diff
@@ -0,0 +1,152 @@
+Especially at the time, with the majority of people having slow and unreliable internet access, the process of downloading large files could, and usually did, take a very long time.
+
+Cohen would leave MojoNation in 2001 and begin work on the open protocol that would become known as [BitTorrent](https://en.wikipedia.org/wiki/BitTorrent).
+BitTorrent would immediately become popular with the Linux community, as it allowed for the free distribution of Linux distributions and other free software that would not have had the funding or labor to distribute their software otherwise.
+And by 2003, [ThePirateBay](https://en.wikipedia.org/wiki/The_Pirate_Bay) would launch, becoming one of the most infamous websites in the world to download free _pirated_ content, such as music, movies, and TV shows.
+
+## BitTorrent Basics
+
+BitTorrent decentralizes the file sharing process. 
+Rather than one server being required to serve a file to many users, which can become bottle necked, BitTorrent allows for many users to share files with each other by joining a "swarm".
+Users within the swarm can advertise they have a file, or a part of it, and exchange "pieces" with other users in the swarm.
+
+![BitTorrent Swarm](./img/tracker-light-mode.svg#gh-light-mode-only)![BitTorrent Swarm](./img/tracker-dark-mode.svg#gh-dark-mode-only)
+
+### Torrent files
+
+In order to distribute a file using BitTorrent, a ["torrent"](https://en.wikipedia.org/wiki/Torrent_file) file is created, which contains metadata about the file(s) to be shared.
+You can use a tool such as [TorrentEditor](https://torrenteditor.com/) to view the contents of a torrent file online. Here for example is the contents of the Ubuntu 23.04 Server ISO.
+
+:::info
+
+#### Ubuntu 23.04 Live Server Torrent Details
+
+**Filename:** ubuntu-23.04-live-server-amd64.iso.torrent
+
+##### Hash Details
+
+Info Hash: D6B4535B A8F2B340 12BC6335 69F113E7 7017E032
+
+##### Tracker Information
+
+**Tracker URL:** https://torrent.ubuntu.com/announce
+
+**Seeds:** 158, **Peers:** 4, **Completed:** 1
+
+Additional Tracker: https://ipv6.torrent.ubuntu.com/announce
+
+**Seeds:** 158, **Peers:** 4 **Completed:** 1
+
+##### Metadata
+
+- **Created On:** 04/20/23 12:14:49
+- **Created By:** mktorrent 1.1
+- **Comment:** Ubuntu CD releases.ubuntu.com
+- **Piece Length\*:** 256 KB
+- **Private\*:** Off - Unset
+
+##### Files
+
+**Filename:** ubuntu-23.04-live-server-amd64.iso
+
+**File Size:** 2.46 GB
+:::
+
+The torrent file itself is incredibly small, for this 2.46 GB ISO the torrent itself is only 200 KB.
+
+### Magnet links
+
+The data inside a torrent file is so small, that it can, in fact be can be represented as a URL rather than a file. 
+A ['Magnet link'](https://en.wikipedia.org/wiki/Magnet_URI_scheme) is a URL scheme that allows you to share information about a torrent as a link, rather than a file.
```

Comment:
[Nitpick] Explaining the data and defining Magnet Pick can make it difficult for users to understand at first glance. I suggest defining the term first and then describing its connection to the display of a torrent's data.  
```suggestion
The data inside a torrent file is so small, that it can, in fact be can be represented as a URL rather than a file. 
A ['Magnet link'](https://en.wikipedia.org/wiki/Magnet_URI_scheme) is a URL scheme that allows you to share information about a torrent as a link, rather than a file.
```